bin/plugin/commands/release.js

```diff
@@ -73,32 +73,48 @@ async function runReleaseBranchCreationStep(
 	await runStep( 'Creating the release branch', abortMessage, async () => {
 		const packageJsonPath = gitWorkingDirectoryPath + '/package.json';
 		const packageJson = readJSONFile( packageJsonPath );
-		const nextMajor = getNextMajorVersion( packageJson.version );
-		const parsedMajorVersion = semver.parse( nextMajor );
+		const packageVersion = semver.parse( packageJson.version );
+		const isPackageVersionRC = packageVersion.prerelease.length > 0;
+
+		// if the last tag is an RC then we can assume that we should generate a new rc with the same version number
+		if ( isPackageVersionRC ) {
+			version = semver.inc( packageVersion.version, 'prerelease', 'rc' );
+		} else {
+			const nextMajor = getNextMajorVersion( packageJson.version );
+			version = nextMajor + '-rc.1';
+		}
+
+		const parsedVersion = semver.parse( version );
 
-		version = nextMajor + '-rc.1';
+		versionLabel = version.replace( /\-rc\.([0-9]+)/, ' RC$1' );
 		releaseBranch =
-			'release/' +
-			parsedMajorVersion.major +
-			'.' +
-			parsedMajorVersion.minor;
-		versionLabel = nextMajor + ' RC1';
+			'release/' + parsedVersion.major + '.' + parsedVersion.minor;
 
-		await askForConfirmation(
-			'The Plugin version to be used is ' +
-				formats.success( version ) +
-				'. Proceed with the creation of the release branch?',
-			true,
-			abortMessage
-		);
+		if ( ! isPackageVersionRC ) {
+			await askForConfirmation(
+				'The Plugin version to be used is ' +
+					formats.success( version ) +
+					'. Proceed with the creation of the release branch?',
+				true,
+				abortMessage
+			);
 
-		// Creating the release branch
+			log(
+				'>> The local release branch ' +
+					formats.success( releaseBranch ) +
+					' has been successfully created.'
+			);
```

Comment:
Shouldn't this remain after the call to `git.createLocalBranch`?

Reply:
Good point. The call to `createLocalBranch` has been moved down becuase I found out that was also needed for the RC scenario, as it essentially checkout th branch. It doesn't make sense indeed to show a confirmation message before the actual thing happened :)